interface/controllers/monster_controller.go

```diff
@@ -0,0 +1,27 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/Le-MaliX/ACADEMY-GO-Q42021/services"
+	"github.com/gin-gonic/gin"
+)
+
+func GetAllMonsters(c *gin.Context) {
+	monsters, err := services.GetAllMonsters()
+	if err != nil {
+		c.JSON(http.StatusBadRequest, c.Error(err))
+		return
+	}
+	c.JSON(http.StatusOK, monsters)
+}
+
+func GetMonsterById(c *gin.Context) {
+	id := c.Param("id")
```

Comment:
Since the ID fields are integers on your DB(the csv file), I would suggest working with the actual data type, which would imply some casting which is fine.

This would imply validating that the ID that you receive here is actually an integer. All the validations of the info you receive on the requests should be on the controller layer of your apps